bookshelf/templates/view.html

```diff
@@ -0,0 +1,55 @@
+{#
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#}
+
+{% extends "base.html.twig" %}
+
+{% block content %}
+
+<h3>Book</h3>
+<form method="post" action="/books/{{book.id}}/delete" id="deleteForm">
+  <div class="btn-group">
+    <a href="/books/{{book.id}}/edit" class="btn btn-primary btn-sm">
+      <i class="glyphicon glyphicon-edit"></i>
+      Edit book
+    </a>
+    <button id="submit" type="submit" class="btn btn-danger btn-sm">
+      <i class="glyphicon glyphicon-trash"></i>
+      Delete book
+    </button>
+  </div>
+</form>
+
+<div class="media">
+  {# [START book_image] #}
+  <div class="media-left">
+    {% if book.imageUrl %}
+      <img class="book-image" src="{{book.imageUrl}}">
+    {% else %}
+      <img class="book-image" src="http://placekitten.com/g/128/192">
```

Comment:
I would rather see this:

```html
<img class="book-image" src="{{ book.imageUrl ?: 'http://placekitten.com/g/128/192' }}">
```

Or even better, set the default to a var, so it's `book.imageUrl ?: defaultImage`